test/napi/src/js/typedarrays.rs

```diff
@@ -8,6 +8,18 @@ pub fn return_array_buffer(mut cx: FunctionContext) -> JsResult<JsArrayBuffer> {
     Ok(b)
 }
 
+pub fn return_array_buffer_from_slice(mut cx: FunctionContext) -> JsResult<JsArrayBuffer> {
+    let len: Handle<JsNumber> = cx.argument(0)?;
+    let len: f64 = len.value(&mut cx);
+    let len: u32 = len as u32;
+    let mut v: Vec<u8> = Vec::new();
+    for i in 0..len {
+        v.push(i as u8);
+    }
+    let b: Handle<JsArrayBuffer> = JsArrayBuffer::from_slice(&mut cx, &v)?;
+    Ok(b)
+}
```

Comment:
FYI, this can be simplified a bit with an iterator collect.

```suggestion
pub fn return_array_buffer_from_slice(mut cx: FunctionContext) -> JsResult<JsArrayBuffer> {
    let len = cx.argument::<JsNumber>(0)?.value(&mut cx) as usize;
    let v = (0..len).map(|i| i as u8).collect::<Vec<_>>();

    JsArrayBuffer::from_slice(&mut cx, &v)
}
```